x-pack/plugins/cases/public/components/add_comment/index.tsx

```diff
@@ -66,10 +68,12 @@ export const AddComment = React.memo(
       { id, caseId, onCommentPosted, onCommentSaving, showLoading = true, statusActionButton },
       ref
     ) => {
+      const storage = useMemo(() => new Storage(window.sessionStorage), []);
```

Comment:
Let's use [`useSessionStorage`](https://github.com/streamich/react-use/blob/master/docs/useSessionStorage.md) from the [`react-use`](https://github.com/streamich/react-use) library. The hook does not export a `remove` function so we need to set the value to an empty string.

Reply:
fixed